tests/test_models/test_amenity.py

```diff
@@ -79,12 +79,12 @@ def test_name_attr(self):
     def test_to_dict_creates_dict(self):
         """test to_dict method creates a dictionary with proper attrs"""
         am = Amenity()
-        print(am.__dict__)
+        # print(am.__dict__)
         new_d = am.to_dict()
         self.assertEqual(type(new_d), dict)
         self.assertFalse("_sa_instance_state" in new_d)
         for attr in am.__dict__:
-            if attr is not "_sa_instance_state":
```

Comment:
my mistake !!!please delete if attr is not "_sa_instance_state": and keep if attr != "_sa_instance_state":